diff --git a/main.go b/main.go
index f3e09e1..415e205 100644
--- a/main.go
+++ b/main.go
@@ -26,7 +26,9 @@ import (
 	"strconv"
 	"strings"
 	"syscall"
+	"unsafe"
 
+	"github.com/alecthomas/kingpin/v2"
 	"github.com/go-kit/log/level"
 	"github.com/pkg/errors"
 	"github.com/prometheus/client_golang/prometheus"
@@ -36,13 +38,16 @@ import (
 	"github.com/prometheus/common/version"
 	"github.com/prometheus/exporter-toolkit/web"
 	webflag "github.com/prometheus/exporter-toolkit/web/kingpinflag"
-	"gopkg.in/alecthomas/kingpin.v2"
+	gopkg_kingpin "gopkg.in/alecthomas/kingpin.v2"
 	"gopkg.in/yaml.v3"
 
 	"github.com/prometheus/blackbox_exporter/config"
 	"github.com/prometheus/blackbox_exporter/prober"
 )
 
+var gopkg_kingpin_CommandLine =
+	(*gopkg_kingpin.Application)(unsafe.Pointer(kingpin.CommandLine))
+
 var (
 	sc = &config.SafeConfig{
 		C: &config.Config{},
@@ -68,7 +73,7 @@ func main() {
 func run() int {
 	kingpin.CommandLine.UsageWriter(os.Stdout)
 	promlogConfig := &promlog.Config{}
-	flag.AddFlags(kingpin.CommandLine, promlogConfig)
+	flag.AddFlags(gopkg_kingpin_CommandLine, promlogConfig)
 	kingpin.Version(version.Print("blackbox_exporter"))
 	kingpin.HelpFlag.Short('h')
 	kingpin.Parse()
