diff --git a/generator/main.go b/generator/main.go
index 464908e..8188bbe 100644
--- a/generator/main.go
+++ b/generator/main.go
@@ -104,8 +104,8 @@ var (
 	snmpMIBOpts        = kingpin.Flag("snmp.mibopts", "Toggle various defaults controlling MIB parsing, see snmpwalk --help").Default("e").String()
 	generateCommand    = kingpin.Command("generate", "Generate snmp.yml from generator.yml")
 	userMibsDir        = kingpin.Flag("mibs-dir", "Paths to mibs directory").Default("").Short('m').Strings()
-	generatorYmlPath   = generateCommand.Flag("generator-path", "Path to the input generator.yml file").Default("generator.yml").Short('g').String()
-	outputPath         = generateCommand.Flag("output-path", "Path to write the snmp_exporter's config file").Default("snmp.yml").Short('o').String()
+	generatorYmlPath   = generateCommand.Flag("generator-path", "Path to the input generator.yml file").Default("/etc/prometheus/snmp-generator.yml").Short('g').String()
+	outputPath         = generateCommand.Flag("output-path", "Path to write the snmp_exporter's config file").Default("/etc/prometheus/snmp.yml.new").Short('o').String()
 	parseErrorsCommand = kingpin.Command("parse_errors", "Debug: Print the parse errors output by NetSNMP")
 	dumpCommand        = kingpin.Command("dump", "Debug: Dump the parsed and prepared MIBs")
 )
