diff --git a/nut_exporter.go b/nut_exporter.go
index 1399f5b..d00be57 100644
--- a/nut_exporter.go
+++ b/nut_exporter.go
@@ -5,6 +5,7 @@ import (
 	"net/http"
 	"os"
 	"strings"
+	"unsafe"
 
 	"github.com/alecthomas/kingpin/v2"
 	"github.com/go-kit/log/level"
@@ -15,10 +16,14 @@ import (
 	"github.com/prometheus/common/version"
 	"github.com/prometheus/exporter-toolkit/web"
 	"github.com/prometheus/exporter-toolkit/web/kingpinflag"
+	gopkg_kingpin "gopkg.in/alecthomas/kingpin.v2"
 
 	"github.com/DRuggeri/nut_exporter/collectors"
 )
 
+var gopkg_kingpin_CommandLine =
+	(*gopkg_kingpin.Application)(unsafe.Pointer(kingpin.CommandLine))
+
 var Version = "testing"
 
 var (
@@ -133,7 +138,7 @@ func (h *metricsHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 
 func main() {
 	promlogConfig := &promlog.Config{}
-	flag.AddFlags(kingpin.CommandLine, promlogConfig)
+	flag.AddFlags(gopkg_kingpin_CommandLine, promlogConfig)
 	kingpin.Version(Version)
 	kingpin.HelpFlag.Short('h')
 	kingpin.Parse()
