diff --git a/nut_exporter.go b/nut_exporter.go
index 3eb7cb9..aa8ed4f 100644
--- a/nut_exporter.go
+++ b/nut_exporter.go
@@ -10,10 +10,10 @@ import (
 	"github.com/alecthomas/kingpin/v2"
 	"github.com/go-kit/log/level"
 	"github.com/prometheus/client_golang/prometheus"
+	versioncollector "github.com/prometheus/client_golang/prometheus/collectors/version"
 	"github.com/prometheus/client_golang/prometheus/promhttp"
 	"github.com/prometheus/common/promlog"
 	"github.com/prometheus/common/promlog/flag"
-	"github.com/prometheus/common/version"
 	"github.com/prometheus/exporter-toolkit/web"
 	"github.com/prometheus/exporter-toolkit/web/kingpinflag"
 
@@ -79,7 +79,7 @@ var collectorOpts collectors.NutCollectorOpts
 var logger = promlog.New(&promlog.Config{})
 
 func init() {
-	prometheus.MustRegister(version.NewCollector(*metricsNamespace))
+	prometheus.MustRegister(versioncollector.NewCollector(*metricsNamespace))
 }
 
 type metricsHandler struct {
